test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs

```diff
@@ -210,30 +216,51 @@ protected void CreateModel(string modelUrl)
             // You can apply camera properties if the camera parameter exists.
             if (model.GetCameraCount() > 0u)
             {
-                additionalCamera = new Tizen.NUI.Scene3D.Camera();
-                // If we success to make additional camera from model, Add that camera into sceneview, and select that.
-                if (model.ApplyCamera(0u, additionalCamera))
+                additionalCameraList = new List<Tizen.NUI.Scene3D.Camera>();
+                bool firstSuccessedCamera = true;
```

Comment:
very minor point :
```suggestion
                bool firstSucceededCamera = true;
```

Reply:
Thank you for your review. But due to the security policy, I cant moify this patch in office. I'll make another fixup patch soon